vault/identity_store_util.go

```diff
@@ -150,6 +150,7 @@ func (i *IdentityStore) loadGroups(ctx context.Context) error {
 			for _, memberEntityID := range group.MemberEntityIDs {
 				entity, err := i.MemDBEntityByID(memberEntityID, false)
 				if err != nil {
+					txn.Abort()
```

Comment:
Why is this necessary?  There are no writes to the transaction within this for loop that I can see.  I mean, it's good hygiene, but it doesn't actually fix a bug, does it?

Reply:
A few lines above, there is `txn := i.db.Txn(true)`. When `txn` is opened for write, it stops the world until it is released back.